tests/testthat/test-Model-methods.R

```diff
@@ -4,21 +4,72 @@
 
 test_that("doseFunction-GeneralModel returns correct dose function", {
   model <- h_get_logistic_log_normal()
-  samples <- h_as_samples(list(alpha0 = 1, alpha1 = 2), burnin = 10000, fixed = FALSE)
-  dose_args <- c("x", "model", "samples")
+  samples <- Samples(list(alpha0 = 1, alpha1 = 2), options = McmcOptions(samples = 1))
 
   dose_fun <- doseFunction(model, alpha0 = 1, alpha1 = 2)
-  dose_fun_dose_args <- as.character(body(dose_fun)[[2]][-1])
   dose_fun_env <- environment(dose_fun)
 
+  expect_equal(formalArgs(doseFunction), c("model", "..."))
```

Comment:
This can do a bit easier with the `expect_function()` from checkmate

Reply:
Or did you try that already ?

Reply:
Actually, I think there is no need to check is the `doseFunction` is a function, I mean - this is the main function we want to test to there is no way `doseFunction` is not a function? It is like for instance `dose` function - we do not test if this is a function, but rather we test its results. What do you think @danielinteractive ?

Reply:
ah my point is that with that expectation function you can check the arguments a bit easier

Reply:
Ah, sure - I will do that then. Tx!